### PR TITLE
Update search-mdn extension

### LINKED ISSUE #12921: [Search MDN] ...
### Extension

https://www.raycast.com/krzysztofzuraw/search-mdn

### Raycast Version

1.76.0

### macOS Version

14.5

#### Error:
```
Error: Cannot parse render tree JSON:
Invalid escape sequence around line 1, column 2696.)
```

### Steps To Reproduce

1. Search MDN for Array.sort
2. Choose Array.prototype.sort

### Current Behaviour

Shows something went wrong screen

### Expected Behaviour

Load documentation about Array.sort

### PR DESCRIPTION
**Pull request by j3lte**
## Description

- This should fix #12921 
- It's a bit of dirty fix. Basically we're trying to escape the `\u` characters. Here's the result (until Raycast can handle this properly):

Screenshot 2024-09-02 at 16 30 12
_(See the `\u` characters)_

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/search-mdn/src/hooks/use-result.ts [hunk 1]
```diff
@@ -71,6 +72,12 @@ const cleanupContent = (raw: string, res: Result): string => {
     return `[${text}](${url})`;
   });
 
+  // Unescape any unicode characters (because all \u characters should be treated as code)
+  // It's a dirty fix, but the Markdown parser doesn't seem to handle \u characters well
```
Comment (jfkisafk) [inline extensions/search-mdn/src/hooks/use-result.ts:76]: (non-blocking) Thank you for the fix, should we / have we created a bug/feature request for Raycast to fix their markdown rendering in Detail?

File: extensions/search-mdn/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Updated - 2024-09-02
```
Comment (jfkisafk) [inline extensions/search-mdn/CHANGELOG.md:3]:
```suggestion
## Updated - {PR_MERGE_DATE}
```

**Review by jfkisafk — APPROVED** [review]

File: extensions/search-mdn/src/hooks/use-result.ts (hunk 1 above)
Comment (j3lte) [reply extensions/search-mdn/src/hooks/use-result.ts:76]: This was already mentioned in the original issue linked above... Also in the other issue that is referring the mentioned issue

Comment (jfkisafk) [reply extensions/search-mdn/src/hooks/use-result.ts:76]: cool cool I noticed it. Tyty. We can resolve this thread.

**Review by jfkisafk — APPROVED** [review]

**Review by krzysztofzuraw — APPROVED** [review]

**Comment by bxb100** [comment]
as I mentioned here #14294, I think it's Raycast API bug

**Comment by j3lte** [comment]
> as I mentioned here #14294, I think it's Raycast API bug

@bxb100 it is indeed a Raycast bug, I'm doing a bit of a workaround

**Comment by pernielsentikaer** [comment]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**